llm_demo/evaluation/eval_golden.py

```diff
@@ -0,0 +1,281 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, Field
+
+
+class EvalData(BaseModel):
+    category: Optional[str] = Field(default=None, description="evaluation category")
+    query: Optional[str] = Field(default=None, description="user query")
+    instruction: Optional[str] = Field(
+        default=None, description="instruction to llm system"
+    )
+    content: Optional[str] = Field(default=None)
+    tool_calls: Optional[List[Dict[str, Any]]] = Field(default=None)
```

Comment:
is "List[Dict[str, Any]]" the best option here? Or would ie be better to introduce another model like "ToolCall" or "ToolRecord" and have "List[ToolCall]"?